source/_faq/unique_id.markdown

```diff
@@ -4,10 +4,21 @@ description: "This entity does not have a unique ID?"
 ha_category: Configuration
 ---
 
-When you see
+If you try to access the configuration dialog for an entity in your Home Assistant, you might end up seeing this message:
 
-```text
-This entity does not have a unique ID, therefore its settings cannot be managed from the UI
-```
+<p class='img'>
+<img src='/images/faq/faq_no_unique_id.jpg' alt='Screenshot of popup for no unique ID'>
+</p>
 
-this simply means that you can't change the `entity_id` or the friendly name from the UI. You can still use the _Customizations_ menu to customize that entity. Typically you'll see this when you create entities using YAML.
+This simply means that this entity does not have a unique identification e.g., a serial number or another identifier that is guaranteed to be static and never change\*. As a result, the normal editing process that allows you to change various settings through the user interface (such as the entity ID, icon, friendly name, etc.) is not possible here. 
```

Comment:
Avoid the use of `simply`, it can be perceived as condescending by the reader.


```suggestion
This means that this entity does not have a unique identification, e.g., a serial number or another identifier that is guaranteed to be static and never changes. As a result, the normal editing process that allows you to change various settings through the user interface (such as the entity ID, icon, friendly name, etc.) is not possible here. 
```